### DISCUSSION TITLE
Can customers have accounts that saves order history, address, etc?

### DISCUSSION THREAD

**Discussion by austriker27** [statamic-rad-pack/shopify#66] [Help] [answered]
From what I can, it doesn't seem there's the ability for customers to create accounts so they can see their order history, save addresses, etc. 

This would be a great feature if possible!

**Comment by jackabox** [comment, marked as answer]
Hey,

It's not currently implemented but this would be handled through the frontend using the Storefront API - [https://shopify.dev/docs/storefront-api/reference/customers/customer]().

- You'd generate a customer access token by sending the email/password to the server.
- Then if it returns the access token, you'd pass that to get the customer and their orders.

You also have other endpoints for creating, password reset, etc.

If you didn't want to do it this way, you could modify the Shopify store on a subdomain and link to that.

I do want to get an example up for accounts, but at the minute I'm focused on client work, and cannot spare the resource. Happy for you to run anything by me though if you want a hand :)

**Reply by austriker27** [reply]
No worries, I appreciate the information! That info is super helpful for me to dive into if I decide to add this now to my site. Thanks a lot!
